tidb-configuration-file.md

```diff
@@ -46,6 +46,10 @@ The TiDB configuration file supports more options than command-line parameters.
 
 ### `oom-action`
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
```

Comment:
This is not aligned with [docs-cn changes](https://docs.pingcap.com/zh/tidb/v4.0/tidb-configuration-file#oom-action). @jackysp

Reply:
done